modules/contact/test/tests/3d-mortar-contact/tests

```diff
@@ -83,7 +83,7 @@
                   'using nodal-attached geometry and a frictional pressure vector generated by an '
                   'auxiliary kernel through a user-friendly action. Results are diffed against '
                   'non-action output'
-    rel_err = 3.0e-5
+    rel_err = 1.0e-4
```

Comment:
Is this solving the exact same problem as the non-action test? It would seem weird to have to have a looser tolerance

Reply:
Yeah, it is and it is odd. I  am making the same changes to all 3D frictional problems.